analysers/analyser_osmosis_roundabout_reverse.py

```diff
@@ -67,10 +68,12 @@ def __init__(self, config, logger = None):
             self.driving_side = ""
```

Comment:
Also need the definition to be put under a if

Reply:
Done